app/controllers/orders_controller.rb

```diff
@@ -9,7 +9,7 @@ def create
     if @order.save
       session[:cart] = nil
 
-      flash[:message] = "Successfully submitted your order totaling #{@order.total_price}"
+      flash[:message] = "Successfully submitted your order totaling $#{@order.total_price}0"
```

Comment:
Do we want to hardcode that 0 in?  I'm worried this could interfere with our production data since if a price total is say, $45.83, this controller would still add a 0 to the end of it.  Can we use a number_to_currency method in a controller?  I don't know the answer to that.